js/pages/tournamentlobbypage.ts

```diff
@@ -710,15 +710,15 @@ export class TournamentLobbyPage extends PageBase {
         /* tslint:disable:no-string-literal */
         const numericTextBinding = ko.bindingHandlers["numericText"] as any;
         /* tslint:enable:no-string-literal */
-        const joinAmountString = numericTextBinding.withCommas(joinAmount.toFixed(0));
+        const joinAmountString = numericTextBinding.withCommas(joinAmount.toFixed(0), ",");
```

Comment:
Add test

Reply:
Probably better would be move `withCommas` to separate function, does not related to binding. That function could be covered by tests and much more cleanly used.